tests/test_api.py

```diff
@@ -149,3 +150,23 @@ def test_chart_line(self, grep):
         self.assertIn(
             b'<svg xmlns:xlink="http://www.w3.org/1999/xlink', resp.get_data()
         )
+
+    def test_healthz_liveness(self):
+        """Test the /healthz/live check endpoint"""
+        result = self.client.get("/healthz/live")
+        assert result.status_code == 200
+        assert result.data == b"OK\n"
+
+    def test_healthz_readiness_ok(self):
+        """Test the /healthz/ready check endpoint"""
+        result = self.client.get("/healthz/ready")
+        assert result.status_code == 200
+        assert result.data == b"OK\n"
+
+    def test_healthz_readiness_not_ok(self):
+        """Test the /healthz/ready check endpoint when not ready"""
+        with mock.patch("datagrepper.app.dm.session") as grepsesh:
```

Comment:
You're mocking the wrong function here again. You don't use `dm.session` in `readiness`.